tests/queries/0_stateless/02345_partial_sort_transform_optimization.sql

```diff
@@ -1,3 +1,3 @@
--- Regression for PartialSortingTransform optimization
--- that requires at least 1500 rows.
-select * from (select * from (select 0 a, toNullable(number) b, toString(number) c from numbers(1e6)) order by a desc, b desc, c limit 1500) limit 10;
+-- Regression for PartialSortingTransform optimization that requires at least 1500 rows.
+SELECT * FROM (SELECT * FROM (SELECT 0 a, toNullable(number) b, toString(number) c FROM numbers(1e6)) ORDER BY a DESC, b DESC, c LIMIT 1500) limit 10;
+SELECT number FROM (SELECT number, 1 AS k FROM numbers(100000) ORDER BY k ASC, number DESC LIMIT 1025, 1023) LIMIT 5;
```

Comment:
@kitaisreal just out of curiosity, what this query checks that was not covered by the first query?

Reply:
It does not cover any new paths. Just to check that with constant column it will work as expected.